source/draft_users.rst

```diff
@@ -0,0 +1,177 @@
+.. _draft_users:
+
+=====================
+Draft User Management
+=====================
+
+Users are managed primarily through the API, by admin users.
+
+.. contents::
+
+-----------
+
+Users Model
+-----------
+
+For reference of the structure of this table, see the :ref:`draft_model<draft_model>` docs.
+
+===========  ========  =============  ==============================  =============================================
+   Name        Type    POST Required           Description                                Notes
+===========  ========  =============  ==============================  =============================================
+username     string        true       Username the user logs in with  Used for display if displayname is null
+password     string        true       Password for user login         Stored as bcrypt hash; not returned on get
+displayname  string        false      Name used for public display
```

Comment:
Should this be ``display_name`` to be consistent with other snake case fields like ``created_at``?